tests/e2e/e2e_test.go

```diff
@@ -333,6 +334,83 @@ func (s *IntegrationTestSuite) TestIBCWasmHooks() {
 	)
 }
 
+func (s *IntegrationTestSuite) TestPacketForwarding() {
```

Comment:
Can we add some code comments to this for what chain were doing an action on, and what chain were querying at different points in this?

Reply:
Added comments